src/api/app/jobs/consistency_check_job.rb

```diff
@@ -74,7 +74,7 @@ def fix
   end
 
   def init
-    User.session ||= User.get_default_admin
+    User.session = User.get_default_admin
```

Comment:
What does this have to do with `User.run_as`?

Reply:
well, not so much - more with leaking sessions :)